manifests/r/RegsoftsSoftware/RegistryRepair/4.5/RegsoftsSoftware.RegistryRepair.locale.en-US.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RegsoftsSoftware.RegistryRepair
+PackageVersion: "4.5"
+PackageLocale: en-US
+Publisher: RegSofts Software
+PublisherUrl: https://www.regsofts.com/free_registry_repair/registry_repair.htm
+PublisherSupportUrl: https://www.regsofts.com/about.htm
+Author: Regsofts
+PackageName: RegpairSetup.exe

```

```suggestion
PackageName: Free Window Registry Repair
```
This should match Control Panel